wiki/Community/id.md

```diff
@@ -0,0 +1,33 @@
+# Komunitas
+
+Berikut ini merupakan daftar artikel yang membahas seputar cara komunikasi yang terjadi di komunitas osu!. Jika kamu mencari cara berkontribusi tetapi belum tahu harus mulai dari mana, kamu dapat membaca artikel [Bagaimana kamu dapat membantu!](/wiki/Community/How_you_can_help!) untuk mengetahui cara berkontribusi.
+
+## Tempat
+
+<!-- TODO: add forums -->
+
```

Comment:
small change in https://github.com/ppy/osu-wiki/pull/6523

```suggestion
- [Forum](/wiki/Community/Forum)
```

Reply:
Sure